### ISSUE TITLE
Prices with commas is parsed incorrectly and results in incorrect number saved.

### ISSUE THREAD

**Issue by lukeholder** [craftcms/commerce#442]
### Description
Hello,

with the last Commerce update a serious bug about the product prices appeared. I made a screen capture: http://recordit.co/0iSAr0idKd

When I enter the price of a product, it needs to be with a comma. After saving, it is shown with a period. When I save it again, the price is multiplied by 100.

I use German as localization

### Steps to reproduce

1. http://recordit.co/0iSAr0idKd

**Comment by nordentwickler** [comment]
yep, i can confirm that bug: https://nordentwickler.de/price.gif

after saving a price without comma, it is going to be added and some zeros too ^^

**Comment by tomdeleu** [comment]
I also can confirm this bug (Dutch localization) 

Any news when this one would be fixed and available in a next version? 
(Don't want to to pressure but it is keeping one of our clients from inputting data in a dev site, so need to be able to reschedule some things.)

**Comment by lukeholder** [comment]
Should have been fixed in beta 8.1

**Comment by tomdeleu** [comment]
Yes, just tried it and seems to work fine. Awesome, thank you!

**Comment by nordentwickler** [comment]
Done, thank you!
